src/test/java/software/amazon/awssdk/crt/test/CredentialsProviderTest.java

```diff
@@ -213,8 +213,7 @@ public void testCreateDestroyProfile_InvalidProfile() throws IOException {
         ProfileCredentialsProvider.Builder builder = ProfileCredentialsProvider.builder()
                 .withConfigFileNameOverride(confPath.toString()).withCredentialsFileNameOverride(credsPath.toString());
 
-        try {
-            builder.build();
+        try (ProfileCredentialsProvider provider = builder.build()) {
```

Comment:
HA

Reply:
https://github.com/awslabs/aws-crt-java/pull/359/files#diff-7a1d883433da59ad94de2b6989b5ae4042739a36ab35dbbd80baf1c2ccf52d71R68-R71 this is actually another fix for it

Reply:
This is the correct fix, should always use try-with-resource if we can